GOVERNANCE.md

```diff
@@ -168,6 +168,30 @@ maintainers—the core maintainers of containerd will not be appended to each su
 Please add the suggested text from our [Non-core project documents](./README.md#non-core-project-documents)
 section to your `README.md`.
 
+### Changing the status of non-core subprojects
+
+A change of status of a non-core subproject can be made by submitting a change of
+status proposal via public forum. The types of changes may include:
+- promoting a non-core subproject to core
+- removing/archiving the non-core subproject
+- merging into another project
+
+The proposal should include some general information on support, releases,
+stability, downstream impacts, steps necessary to effect the change, and any
+additional detail useful for the containerd maintainers to understand the scope and
+nature of the project status change. GitHub issue is the easiest way to provide
+this proposal.
+
+The existing maintainers are given fourteen days to discuss the project change,
+raise objections and cast their vote. Project changes of this type must be approved
+by 2/3 of the current core committers and core committers are encouraged to consider the opinions and objections of the existing non-core project maintainers.
+
+Project status changes do not automatically change the status of any non-core project maintainers.  Project status change proposals may be accompanied by additional proposals to change individual maintainer status, subject to the existing rules set forth in this document.
```

Comment:
nit: wrapping is not right here

Reply:
yeah md doesn't care, the view gets updated when it is loaded.. more a convention to insert extra line breaks or not